src/parachain/amm.ts

```diff
@@ -262,27 +263,27 @@ export class DefaultAMMAPI implements AMMAPI {
         return this._getStableLpTokenFromPoolData(poolId.unwrap().toNumber(), basePoolData);
     }
 
-    private async _getStandardPoolReserveBalances<Currency0 extends CurrencyExt, Currency1 extends CurrencyExt>(
-        token0: Currency0,
-        token1: Currency1,
+    private async _getStandardPoolReserveBalances(
+        token0: CurrencyExt,
+        token1: CurrencyExt,
         pairAccount: AccountId
-    ): Promise<[MonetaryAmount<Currency0>, MonetaryAmount<Currency1>]> {
-        // TODO
-        throw new Error("Method not implemented.");
-    }
+    ): Promise<[MonetaryAmount<CurrencyExt>, MonetaryAmount<CurrencyExt>]> {
+        const [token0Balance, token1Balance] = await Promise.all([
+            this.tokensAPI.balance(token0, pairAccount),
+            this.tokensAPI.balance(token1, pairAccount),
+        ]);
+        // TODO: check if transferable or free balance should be used here
+        const token0MonetaryAmount = token0Balance.transferable;
+        const token1MonetaryAmount = token1Balance.transferable;
```

Comment:
@gregdhill Is it correct to use only transferable balance here for the balance of the uniswap pool or should it be something else?

Reply:
It can be either I think (assuming "transferable" is `free - locked` since we never lock) but I would advise you to use the free balance